docs/scale-cluster.rst

```diff
@@ -44,32 +44,38 @@ dropdown menu at the top left to see the desired projects.
 In the menu on the left hand you should see below the divider line an icon or
 icons representing all clusters associated with the currently selected project.
 Here, find the correct cluster and click on the icon. It will expand and show
-links for three pages: *Overview*, *Metrics* and *Settings*.
+links for four pages: *Overview*, *Backups*, *Metrics* and *Preferences*.
 
 .. image:: _assets/img/cluster-dropdown.png
    :alt: Cloud Console projects cluster selection
 
-Click on *Settings*. This will take you to the Cluster Settings page, where
-you can see the current scale unit of the cluster, as well as an information
-box for the current cluster plan and tier. To scale the cluster up or down
-within the permitted range, simply click the *Edit scale unit* button at the
-top right.
+Click on *Preferences*. This will take you to Cluster access page, where you
+can see and edit your credentials. You can also enable Deletion protection 
+of your cluster and create Whitelist of IP addresses or CIDR blocks that are
+allowed to connect to the cluster. 
 
-.. image:: _assets/img/cluster-settings.png
-   :alt: Cloud Console cluster settings
+.. image:: _assets/img/cluster-access.png
+   :alt: Cloud Console cluster access settings
+
+To edit Cluster scale, click on *Cluster scale* in the top bar. This will take
+you to a page where you can see overview of your Cluster plan as well as 
+current cluster configuration.
+
+.. image:: _assets/img/cluster-preferences.png
+   :alt: Cloud Console cluster preferences
 
 
 .. _scale-cluster-instructions:
 
 Scale a cluster
 ===============
 
-Once arrived at the Cluster Settings page, you can scale the cluster you
-selected in three steps. First, click the *Edit scale unit* button in the top
-right.
+Once arrived at the Cluster scale page, you can scale the cluster you
+selected in three steps. First, click the *Edit cluster configuration* button
+in the top right.
 
-Secondly, choose the desired scaling value in the radio button menu that
-appears.
+Secondly, choose the desired scaling value in either the *Storage* or
+*Choose number of nodes* sections.
```

Comment:
Open question: would we call custom storage increases scaling, in the same way we call expanding nodes scaling? Do we need to distinguish horizontal and vertical? I'm not really sure but wonder what you think

Reply:
Good question. It probably could be better called "storage increase" to be simpler. But when someone says "storage scaling" I pretty much imagine the same thing: increasing the storage. To keep it uniform, I'd stick with "scaling". Opinions?

Reply:
no strong opinions. scaling to me represents adding additional nodes to a cluster and has a different mental model to adding storage. I don't know the precise technicalities of increasing storage but I imagine it as a pool of available disk space as opposed to adding additional hard drives, which 'scaling' might suggest. FWIW to me storage increase is the more natural convention.